larray/viewer.py

```diff
@@ -2458,10 +2463,11 @@ def view(obj=None, title=''):
     If no session object or array dictionary is provided
     as argument, all local arrays are loaded in the editor.
 
-    obj : Session or dict of LArray, optional
+    obj : Session, dict of LArray or str, optional
```

Comment:
obj and title arguments descriptions should be the same between edit() and view().